src/NzbDrone.Core/Download/Clients/Transmission/TransmissionProxy.cs

```diff
@@ -273,24 +273,26 @@ private void AuthenticateClient(HttpRequestBuilder requestBuilder, TransmissionS
                 authLoginRequest.SuppressHttpError = true;
 
                 var response = _httpClient.Execute(authLoginRequest);
-                if (response.StatusCode == HttpStatusCode.MovedPermanently)
-                {
-                    var url = response.Headers.GetSingleValue("Location");
 
-                    throw new DownloadClientException("Remote site redirected to " + url);
-                }
-                else if (response.StatusCode == HttpStatusCode.Conflict)
-                {
-                    sessionId = response.Headers.GetSingleValue("X-Transmission-Session-Id");
-
-                    if (sessionId == null)
-                    {
-                        throw new DownloadClientException("Remote host did not return a Session Id.");
-                    }
-                }
-                else
+                switch (response.StatusCode)
                 {
-                    throw new DownloadClientAuthenticationException("Failed to authenticate with Transmission.");
+                    case HttpStatusCode.MovedPermanently:
+                        var url = response.Headers.GetSingleValue("Location");
+
+                        throw new DownloadClientException("Remote site redirected to " + url);
+                    case HttpStatusCode.Forbidden:
+                        throw new DownloadClientException("Failed to authenticate with Transmission. It may be necessary to add your IP address to RPC whitelist.");
```

Comment:
```suggestion
                        throw new DownloadClientException("Failed to authenticate with Transmission. It may be necessary to add your IP address to RPC whitelist.");
```

Should we specify sonarr's/apps's IP?

Reply:
I think narrowing down which IP should be added would be good, not sure what option we have to keep it generic between different arr apps.